src/state/StateNode.java

```diff
@@ -78,10 +78,14 @@ public String stringForProperty(String property) {
         }
 
         StringBuilder sb = new StringBuilder();
-        sb.append("\n");
-        for (String value : state.get(property)) {
-            sb.append(String.format("%s\n", value));
-        }
+        sb.append(
```

Comment:
The `StringBuilder` is no longer necessary. We can just `return String.format(...);` here.